### PR TITLE
Update Flutter Docs to reference Swift SDK methods

### PR DESCRIPTION
**Pull request by nigelfaustino**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [X ] Yes (Flutter SDK Version 3.0.0 Dec 6, 2022)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md [hunk 1]
```diff
@@ -57,48 +57,71 @@ Add the required permissions to your `AndroidManifest.xml` file:
...
 {% subtab SWIFT %}
-Add Appboy SDK import at the top of the `AppDelegate.swift` file:
+Add Braze SDK import at the top of the `AppDelegate.swift` file:
 ```swift
-import Appboy_iOS_SDK
+import BrazeKit
 ```
 
-In the same file, add the following snippet within the `application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool` method and replace the API identifier key with your value:
+In the same file, create the Braze configuration object with the API key and endpoint of your Braze app within the `application:didFinishLaunchingWithOptions` method and replace the API identifier key and endpoint with your value. Create the Braze instance using the configuration, and create a braze static property on AppDelegate for easy access to SDK features:
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md:65]: Tweaked a bit and removed some words:
```suggestion
In the same file, create the Braze configuration object in the `application:didFinishLaunchingWithOptions` method and replace the API key and endpoint with your app's values. Then, create the Braze instance using the configuration, and create a static property on the AppDelegate for easy access:
```

File: _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md [hunk 2]
```diff
@@ -57,48 +57,71 @@ Add the required permissions to your `AndroidManifest.xml` file:
...
+  let braze = Braze(configuration: configuration)
+  AppDelegate.braze = braze
+
+  return true
+}
 ```
 {% endsubtab %}
 {% subtab OBJECTIVE-C %}
-Add Appboy SDK import at the top of the `AppDelegate.m` file:
+Add Braze SDK import at the top of the `AppDelegate.m` file:
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md:88]:
```suggestion
Import `BrazeKit` at the top of the `AppDelegate.m` file:
```

File: _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md [hunk 3]
```diff
@@ -57,48 +57,71 @@ Add the required permissions to your `AndroidManifest.xml` file:
...
 ```objc
-#import "Appboy-iOS-SDK/AppboyKit.h"
+@import BrazeKit;
 ```
 
-In the same file, add the following snippet within the `application:didFinishLaunchingWithOptions` method and replace the API identifier key with your value:
+In the same file, create the Braze configuration object with the API key and endpoint of your Braze app within the `application:didFinishLaunchingWithOptions` method and replace the API identifier key and endpoint with your value. Create the Braze instance using the configuration, and create a braze static property on AppDelegate for easy access to SDK features:
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md:93]:
```suggestion
In the same file, create the Braze configuration object in the `application:didFinishLaunchingWithOptions` method and replace the API key and endpoint with your app's values. Then, create the Braze instance using the configuration, and create a static property on the AppDelegate for easy access:
```

**Review by hokstuff — APPROVED** [review]
Small wording nits - LGTM after!

File: _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md [hunk 4]
```diff
@@ -57,48 +57,71 @@ Add the required permissions to your `AndroidManifest.xml` file:
...
+func application(
+  _ application: UIApplication,
+  didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
+) -> Bool {
+  let configuration = Braze.Configuration(
+    apiKey: "<BRAZE_API_KEY>",
+    endpoint: "<BRAZE_ENDPOINT>"
+  )
+  // Enable logging of general SDK information (e.g. user changes, etc.)
+  configuration.logger.level = .info
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/flutter/flutter_sdk_integration.md:79]: Imo we can remove logger related lines for simplicity + less maintenance burden (same for obj-c example). They can visit the sample app or iOS docs to see something more fleshed out

**Review by Bucimis — APPROVED** [review]

**Review by jocelynzz — APPROVED** [review]

**Pull request merged**